TestNull2

```diff
@@ -0,0 +1,30 @@
+import java.util.LinkedHashSet;
+ 
+public class SimpleLinkedHashSetExample {
+ 
+  public static void main(String[] args) {
+    //create object of LinkedHashSet
+    LinkedHashSet lhashSet = new LinkedHashSet();
+   
+    /*
+      Add an Object to LinkedHashSet using
+      boolean add(Object obj) method of Java LinkedHashSet class.
+      This method adds an element to LinkedHashSet if it is not
+      already present in LinkedHashSet.
```

Comment:
<img width="630" alt="screen_shot_2017-09-05_at_7_06_00_pm" src="https://user-images.githubusercontent.com/1152949/30063909-cfbc46bc-926d-11e7-87de-80b842609e83.png">
Happening due to the long list of apps/actions